ocaml/sdk-gen/csharp/autogen/src/JsonRpc.cs

```diff
@@ -225,6 +241,10 @@ protected virtual T Rpc<T>(string callName, JToken parameters, JsonSerializer se
 
                     using (var responseStream = new MemoryStream())
                     {
+#if (NET462_OR_GREATER || NETSTANDARD2_0_OR_GREATER)
+                        // https://opentelemetry.io/docs/specs/semconv/rpc/json-rpc/
```

Comment:
please add a comment explaining the tag within the source. links can change and not work anymore. If you still think it'd be useful to have a link, I'd use https://archive.org/